actions/src/robocorp/actions/__init__.py

```diff
@@ -8,7 +9,12 @@
 version_info = [int(x) for x in __version__.split(".")]
 
 
-def action(func):
+@overload
+def action(func: Callable, is_consequental: bool = False) -> Callable:
+    ...
+
+
+def action(*args, **kwargs):
     """
     Decorator for actions (entry points) which can be executed by `robocorp.actions`.
```

Comment:
The `actions` docstring should be updated to show docs for the `is_consequential`.

Also, please add an entry do `docs/changelog.md` outlining this change.